e2e/tests/mobile/registered-shopper.spec.js

```diff
@@ -16,98 +24,14 @@ const REGISTERED_USER_CREDENTIALS = generateUserCredentials();
 
 test("Registered shopper can checkout items", async ({ page }) => {
   // Create Account and Sign In
-  await page.goto(config.RETAIL_APP_HOME + "/registration");
-
-  const registrationFormHeading = page.getByText(/Let's get started!/i);
-  await registrationFormHeading.waitFor();
-
-  await page
-    .locator("input#firstName")
-    .fill(REGISTERED_USER_CREDENTIALS.firstName);
-  await page
-    .locator("input#lastName")
-    .fill(REGISTERED_USER_CREDENTIALS.lastName);
-  await page.locator("input#email").fill(REGISTERED_USER_CREDENTIALS.email);
-  await page
-    .locator("input#password")
-    .fill(REGISTERED_USER_CREDENTIALS.password);
-
-  await page.getByRole("button", { name: /Create Account/i }).click();
-
-  await expect(
-    page.getByRole("heading", { name: /Account Details/i })
-  ).toBeVisible();
-
-  await expect(page.getByText(/Email/i)).toBeVisible();
-  await expect(page.getByText(REGISTERED_USER_CREDENTIALS.email)).toBeVisible();
+  await registerShopper({
```

Comment:
Thanks for doing this! Refactoring the reusable steps that are duplicated in multiple steps is a good idea.